docs-xml/manpages/net.8.xml

```diff
@@ -550,6 +550,12 @@ account in server manager first.</para>
 
 <para>Create specified group.</para>
 
+</refsect3>
+<refsect3>
+<title>[ADS] LOOKUP</title>
+
+<para>Find the ADS DC using CLDAP lookup.</para>
```

Comment:
I would use:

Lookup the closest Domain Controller in our domain and retrieve server information about it.

Reply:
@cryptomilk Thanks for comment.
But customers like to have sync between 'net help ads lookup' and 'man net' entries.
Though I agree exact duplicate of 'net help ads lookup' would not be good. But somehow I believe this would be good:
>Lookup the closest Domain Controller in our domain and retrieve server information for same using CLDAP lookup. 

What's your thoughts

Reply:
Use:

`<para>Lookup the closest Domain Controller in our domain and retrieve server information about it.</para>`